sdk/communication/azure-mgmt-communication/setup.py

```diff
@@ -83,6 +83,7 @@
         'msrest>=0.5.0',
         'msrestazure>=0.4.32,<2.0.0',
```

Comment:
Hi @turalf , Thanks for help fixing dependency in mgmt plane. 

I think msrestazure dependency can be removed for track2 package.  would you help remove this line? Thanks!